### PR TITLE
KIWI-1289 template

### PR DESCRIPTION
**Pull request by Caitlin-cooling**
### What changed

Adds BAV template and spec

### Why did it change

Needed for other tickets

### Screenshots

Deploys successfully
image

### Issue tracking
- [KIWI-1289](https://govukverify.atlassian.net/browse/KIWI-1289)

### PII logging

- [x] Verified that no PII data is being logged

[KIWI-1289]: https://govukverify.atlassian.net/browse/KIWI-1289?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/bav-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,197 @@
+openapi: 3.0.1
+info:
+  title:
+     Fn::Sub: "${AWS::StackName} - BAV Credential Issuer Private API"
+  version: "1.0"
+  description: >-
+    Bank Account Verification CRI - Frontend and Backend API specification
```
Comment (ViveakV) [inline deploy/bav-spec.yaml:7]: [nit]: Frontend?

File: deploy/bav-spec.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  contact:
+    name: Darren Dunford
+    email: darren.dunford@digital.cabinet-office.gov.uk
```
Comment (ViveakV) [inline deploy/bav-spec.yaml:10]: I wonder if this should be @mudassar-jafer going forward?

File: deploy/samconfig.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version = 0.1
 [dev]
 [dev.deploy]
 [dev.deploy.parameters]
-stack_name = "bav-api"
+stack_name = "bav-api-cc"
```
Comment (ViveakV) [inline deploy/samconfig.toml:5]: -cc

Comment (Caitlin-cooling) [reply deploy/samconfig.toml]: oops fixed

**Review by ViveakV — APPROVED** [review]

File: deploy/bav-spec.yaml (hunk 2 above)
Comment (Caitlin-cooling) [reply deploy/bav-spec.yaml]: good shout - I've checked with him and he is happy for us to do that so have made the change

**Pull request merged**